client/src/components/BackButton.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import back_btn from "../stories/assets/back_btn.svg";
+import styled from "styled-components/macro";
+
+const PrimaryButton = styled.button`
+  border: none;
+  background: transparent;
+`;
+
+export default function BackButton() {
+  return (
+    <PrimaryButton>
+      <img src={back_btn} className="" alt="back button" />
```

Comment:
If you don't use the class name, it's better to delete it right away :)

Reply:
Thanks marie, this is a placeholder 🙂 it will be filled by today.